app/views/submissions/submit_comment.erb

```diff
@@ -1,14 +1,18 @@
-<form accept-charset="UTF-8" action="/submissions/<%= submission.key %>/nitpick" method="POST">
+<% if current_user.guest? %>
+  <p> You're not logged in right now.Please login via <a href="/login">GitHub </a> to comment </p>
```

Comment:
Will this login link invoke the magic that redirects you back to where you came from once you're logged in?